example/pic2bin.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import binascii
+import zlib
+import cv2
+import numpy as np
+
+ENTER_KEY_WIN = 13
+ENTER_KEY_LINUX = 10
+ESC_KEY = 27
+
+
+def nothing(x):
+    pass
+
+
+if len(sys.argv) <= 1:
+    print("usage: pic2bin.py <imagefile>")
+    sys.exit()
+
+windowname = sys.argv[1] + '    Enter : save    Esc : quit'
```

Comment:
(nits) Use only single-quote or double-quote for string literals.
If you prefer double-quotes, let `black` replace them for you.
```
$ black -l 100 example/pic2bin.py  # this has no -S option and it'll replace single-quotes with double-quotes
```